shop_stripe/models.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+import logging
+import stripe
+from django.db import models
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class StripeCustomer(models.Model):
+	"""
+	A simple model linking a site user with Stripe
+	customer created using stripe API.
+	"""
+
+	class Meta:
+		verbose_name = "Stripe Customer"
```

Comment:
It looks like tabs are being used in this file. These should be replaced with spaces.